src/js/components/Popup/Popup.stories.tsx

```diff
@@ -117,6 +117,25 @@ export const PopupActionsOverlay = () => (
   </div>
 );
 
+export const PopupActionsCustomizedOrigin = () => (
+  <div style={{ display: 'flex', justifyContent: 'center', minHeight: '100px' }}>
+    <div>
+      <PopupAction
+        text={<img src="https://res-4.cloudinary.com/crunchbase-production/image/upload/c_lpad,f_auto,q_auto:eco/v1398359819/ebi2jfu0t7embtj1i35w.png"></img>}
+        actions={actions}
+        direction={['down', 'rightHang']}
+        showOnHover={false}
+        popupStyle={{ width: '150px' }}
+        originButtonStyle={{
```

Comment:
I would recommend to change the prop name to just `buttonStyle`

Reply:
Thanks @jondauz, rename made. 👍